### PR TITLE
Fix for i18n to use defaultMessages instead to default to the id, Upd…

### PR DESCRIPTION
**Pull request by sneridagh**
…ate `babel-plugin-react-intl` to latest

### CODE REVIEW

**Comment by sneridagh** [comment]
@tisto @robgietema https://github.com/formatjs/react-intl/blob/master/docs/Components.md#message-formatting-fallbacks it seems `react-intl` own fallback already provides what the modified line was trying to accomplish. Since we are setting our own fallback, the other was broken. Changed to use only `react-intl` fallback since at the end it will do the same.

File: theme/themes/pastanaga/extras/blocks.less [hunk 1]
```diff
@@ -622,6 +622,10 @@ body.has-toolbar.has-sidebar-collapsed .ui.wrapper > .ui.inner.block.full {
     .icon {
       margin-bottom: 10px !important;
     }
+
+    &.table {
```
Comment (tisto) [inline theme/themes/pastanaga/extras/blocks.less]: @sneridagh committed by accident?

Comment (sneridagh) [reply theme/themes/pastanaga/extras/blocks.less]: all fine, it's a small CSS fix that it was not worth the hassle to create a single PR for it... it's kind of related with the blockchooser

Comment (sneridagh) [reply theme/themes/pastanaga/extras/blocks.less]: I'm in get V4 final asap mode ;)

**Pull request merged**